chain/default_syncer_test.go

```diff
@@ -49,6 +49,8 @@ type DefaultSyncerTestParams struct {
 	minerAddress      address.Address
 	minerOwnerAddress address.Address
 	minerPeerID       peer.ID
+
+	blockTimeTicker func() uint64
```

Comment:
Will likely need something more robust than this moving forward, what that looks like is fuzzy to me at the moment -- for now doing the simple thing.